youtubemodal/index.js

```diff
@@ -184,14 +186,19 @@ YouTubeModal.prototype.onHistoryChange_ = function(e) {
  * Plays a YouTube video.
  * @param {string} videoId Video ID to play.
  * @param {boolean=} opt_updateState Whether to update the history state.
+ * @param {number?} opt_startTime A specific time in the video to start at.
```

Comment:
the `?` means "nullable" whereas `=` means "optional", so switch to `=` here.

ref: https://google.github.io/styleguide/javascriptguide.xml

![screen shot 2018-06-19 at 8 01 10 pm](https://user-images.githubusercontent.com/387282/41635070-97ad4dbe-73fb-11e8-9ec6-cd696e2e253e.jpg)

Reply:
Done